pkgs/data/fonts/kanji-stroke-order-font/default.nix

```diff
@@ -1,19 +1,19 @@
 { stdenv, fetchzip }:
 
 let
-  version = "4.002";
+  version = "4.003";
 in fetchzip {
   name = "kanji-stroke-order-font-${version}";
 
-  url = "https://sites.google.com/site/nihilistorguk/KanjiStrokeOrders_v${version}.zip?attredirects=0";
+  url = "https://drive.google.com/uc?export=download&id=1gd5vUzwgfUSggn8ZbEG2m03x3bbwYPfL";
```

Comment:
Unfortunately, the font is now distributed through Google Drive or Dropbox. I don't know if nixpkgs accept such sources. Should I mirror the font somewhere?

Reply:
It would be better to have a stable and version-controlled source instead of a google drive link that could disappear anytime. If there is no official source repository, maybe you would like to mirror this in a github repo?

Reply:
I think you could the Debian mirror https://salsa.debian.org/fonts-team/fonts-kanjistrokeorders

Reply:
Thank you for your suggestions. I've changed source URL to the Debian mirror.